db/schema.rb

```diff
@@ -66,8 +56,10 @@
   end
 
   create_table "frameworks_frameworks", force: :cascade do |t|
```

Comment:
@nrankin realising the table should have also been called `framework_frameworks` to be consistent with other join tables (eg `measure_categories`

Sadly the same is true for 
- `frameworks_taxonomies` (> `framework_taxonomies`)
- `indicators_recommendations` (> `indicator_recommendations`) 
- `recommendations_recommendations` (> `recommendation_recommendations`)

Reply:
I'm gonna fix these one table at a time, running/fixing the specs as I go so it doesn't get too confusing when everything breaks :)